05-classification.Rmd

```diff
@@ -253,30 +255,38 @@ attrs <- c("Perimeter", "Concavity")
 my_distances <- table_with_distances(cancer[, attrs], new_point)
 neighbours <- cancer[order(my_distances$Distance), ]
 ```
-To predict the label of a new observation, i.e., classify it as either benign
-or malignant, the K-nearest neighbour classifier generally finds the $K$
+
+In order to actually make predictions for new observations in practice, we
+will need a classification algorithm. Here we will use the K-nearest neighbours
+classification algorithm.
+To predict the label of a new observation (here, classify it as either benign
+or malignant), the K-nearest neighbours classifier generally finds the $K$
 "nearest" or "most similar" observations in our training set, and then uses
-their diagnoses to make a prediction for the new observation's diagnosis. To
-illustrate this concept, we will walk through an example.  Suppose we have a
-new observation, with perimeter of `r new_point[1]` and concavity of `r new_point[2]` 
-(labelled in red on the scatterplot), whose diagnosis "Class" is
-unknown.
+their diagnoses to make a prediction for the new observation's diagnosis. $K$ 
+is a number that we must choose; for now, we will assume that someone has chosen
+$K$ for us. We will cover how to choose $K$ ourselves in the next chapter. 
 
+To illustrate the concept of K-nearest neighbours classification, we 
+will walk through an example.  Suppose we have a
+new observation, with perimeter of `r new_point[1]` and concavity of `r new_point[2]`, whose 
+diagnosis "Class" is unknown. This new observation is the red scatter point in
+Figure \@ref(fig:05-knn-1).
 
-```{r 05-knn-1, echo = FALSE, fig.height = 4, fig.width = 5, fig.cap="Scatterplot of concavity versus perimeter with new observation labelled in red"}
+
+```{r 05-knn-1, echo = FALSE, fig.height = 4, fig.width = 5, fig.cap="Scatter plot of concavity versus perimeter with new observation labelled in red"}
 perim_concav +
   geom_point(aes(x = new_point[1], y = new_point[2]), color = cbPalette[6], size = 2.5)
 ```
 </center>
 
-We see that the nearest point to this new observation is **malignant** and
+Figure \@ref(fig:05-knn-2) shows that the nearest point to this new observation is **malignant** and
 located at the coordinates (`r round(neighbours[1, c(attrs[1], attrs[2])],
```

Comment:
```suggestion
Figure \@ref(fig:05-knn-2) shows that the nearest point to this new observation is 
has the diagnosis label of **malignant** 
and located at the coordinates (`r round(neighbours[1, c(attrs[1], attrs[2])],
```

Reply:
disagree; more wordy and doesn't add anything